src/services/edusharing/index.js

```diff
@@ -40,33 +40,31 @@ class EduSharingPlayer {
 	}
 }
 
-class MerlinToken {
+class MerlinTokenService {
 	find(data) {
 		return MerlinTokenGenerator.FIND(data);
 	}
 }
 
 module.exports = (app) => {
 	const eduSharingRoute = '/edu-sharing';
-	const eduSharingPlayerRoute = '/edu-sharing/player';
-	const merlinRoute = '/edu-sharing-merlinToken';
-	const docRoute = '/edu-sharing/api';
-
 	app.use(eduSharingRoute, new EduSharing());
 	const eduSharingService = app.service(eduSharingRoute);
 	eduSharingService.hooks(hooks);
 
+	const eduSharingPlayerRoute = '/edu-sharing/player';
 	app.use(eduSharingPlayerRoute, new EduSharingPlayer(), (req, res) => {
 		res.send(res.data);
 	});
 	const eduSharingPlayerService = app.service(eduSharingPlayerRoute);
 	eduSharingPlayerService.hooks(playerHooks);
 
-	app.use(merlinRoute, new MerlinToken(), (req, res) => {
+	const merlinRoute = '/edu-sharing-merlinToken';
+	app.use(merlinRoute, new MerlinTokenService(), (req, res) => {
 		res.send(res.data);
 	});
 	const merlinService = app.service(merlinRoute);
 	merlinService.hooks(merlinHooks);
 
-	app.use(docRoute, staticContent(path.join(__dirname, '/docs/openapi.yaml')));
+	app.use('/edu-sharing/api', staticContent(path.join(__dirname, '/docs/openapi.yaml')));
```

Comment:
Warrum nicht aus const?

Reply:
Taucht nur einmal auf und ich hatte die die jeweils zum Service gehörenden Sachen sortiert. 
Daher wäre die const genau über dem app.use. Das wirkte unnötig in die Länge gezogen.

Reply:
Ne ist für mich auch Ok.